package.json

```diff
@@ -22,7 +31,14 @@
         "prepare-js:dev": "cd JSDemos && npm i && node ../utils/create-config.js && node ../utils/link-repos.js",
         "prepare-mvc": "node utils/prepare-mvc.js",
         "prepare-netcore": "node utils/prepare-netcore.js",
-        "prepare-all": "npm run prepare-js:dev && npm run prepare-mvc && npm run prepare-netcore"
+        "prepare-all": "npm run prepare-js:dev && npm run prepare-mvc && npm run prepare-netcore",
+        "lint-js": "eslint --ext .js .",
```

Comment:
--ext is '.js' by default https://eslint.org/docs/user-guide/command-line-interface#-ext.
So lint-ts and lint-js tasks look identical. If this is truly better to remove `lint-ts` task at all. And keep `"lint-js": "eslint .",` only.